server/DAO/__tests__/MealtimeDAO.test.js

```diff
@@ -0,0 +1,17 @@
+const MealtimeDAOClass = require('../MealtimeDAO')
+const meal = new MealtimeDAOClass()
+const { MealTimes } = require('../../models/Identifier')
+
+describe('Data access getAllMealtimes function test', () => {
+    it('should return all mealtimes array', () => {
+        const mealtimes = meal.getAllMealtimes()
+        expect(mealtimes).toEqual(MealTimes)
+    })
+})
+
+describe('Data access getMealtime function test', () => {
+    it('should return all mealtimes matching given param', () => {
+        const mealtimesArr = [{value: "morning", type: "mealtime"}, {value: "anytime", type: "mealtime"}]
```

Comment:
Possible refactor:  could move this to a fixture file 
server/DAO/__fixtures__/mealTimes if only used by this file,
or server/__fixtures__/mealTimes if used by more files